webui/src/Route.elm

```diff
@@ -134,15 +131,18 @@ urlFor loc =
                 Reports ->
                     "/reports"
 
-                ShowComment id ->
-                    "/comments/" ++ (toString id)
-
                 Survey ->
                     "/survey"
 
                 SurveyResponses ->
                     "/surveyResponses"
 
+                ShowComment id ->
```

Comment:
Thanks for the cleanups! On this one, I recently tried alphabetical sorting the order of all the terms. In this case, I wonder if we could accomplish both the sorting and grouping the related operations by renaming from ShowComment to CommentShow, and from EditComment to CommentEdit?

Reply:
Good eye! I will update.